Makefile

```diff
@@ -12,7 +12,9 @@ copy:
 	scp service/drempelbox.service ${RPI_HOST}:${RPI_TEMP_PATH}/drempelbox.service
 	scp target/aarch64-unknown-linux-gnu/release/drempelbox ${RPI_HOST}:${RPI_TEMP_PATH}/drempelbox
 
-	ssh ${RPI_HOST} sudo systemctl stop drempelbox
+	if systemctl is-active --quiet service; then \
```

Comment:
did you mean
```suggestion
	if ssh ${RPI_HOST} sudo systemctl is-active --quiet drempelbox; then \
```
?

Reply:
Definitely maybe! Don't know how `drempelbox` got lost but it was surely there in my Makefile at some point. I certainly didn't think about the `ssh` part, however^^ Of course the statement seemed to work, there wasn't any drempelbox service running on my machine :roll_eyes: